CenterForOpenScience/osf.io
# [#PLAT-331]Show DOIs on Withdrawn Registrations

## Purpose
Show the DOI for a withdrawn registration on the tombstone page


## Changes



## QA Notes
1. create a registration
2. create a doi for this registration
3. retract this registration
4. check the page of that retracted registration and you should see the doi information about this retracted registration on this page.


## Side Effects



## Ticket
https://openscience.atlassian.net/browse/PLAT-331
